lib/tasks/styles.js

```diff
@@ -31,6 +31,11 @@ module.exports = function(gulp, config) {
 			allowEmpty: true,
 		})
 
+		var sassOptions = {}
+		if(config.sassNodeModulesAlias) {
+			sassOptions = assign({}, sassOptions, { importer: require('../helpers/sassNodeModulesAliasImporter')(config) })
```

Comment:
We can use `Object.assign`, but it's already imported, so it doesn't really matter.

Reply:
I just tried to keep the conventions from around.